service-monitor/src/main/java/com/yahoo/vespa/service/monitor/internal/SuperModelListenerImpl.java

```diff
@@ -25,7 +25,7 @@
     // and atomically using this monitor.
     private final Object monitor = new Object();
     private final SlobrokMonitorManagerImpl slobrokMonitorManager;
-    private SuperModel superModel;
+    private SuperModel superModel = new SuperModel();
```

Comment:
Actually, this.superModel should be set to the local variable with the same name in start(). Calling e.g. get() before start() would be wrong usage and it may be fine to get a NPE for that (or add explicit check start() has been called).